python/metatensor-operations/metatensor/operations/lstsq.py

```diff
@@ -9,7 +10,12 @@
 )
 
 
-def lstsq(X: TensorMap, Y: TensorMap, rcond, driver=None) -> TensorMap:
+def lstsq(
+    X: TensorMap,
+    Y: TensorMap,
+    rcond: Optional[float] = None,
```

Comment:
I think this was intentionally not set to None by default, because numpy and torch have different behavior here

```suggestion
    rcond: Optional[float],
```